static/app/views/sharedGroupDetails/sharedGroupHeader.tsx

```diff
@@ -33,6 +41,27 @@ function SharedGroupHeader({group}: Props) {
               />
             )}
           </ShortIdWrapper>
+
+          <TimeStamp data-test-id="sgh-timestamp">
+            {'Last seen '}
```

Comment:
add the translation wrapper on text
```suggestion
            {t('Last seen ')}
```